src/config.py

```diff
@@ -9,14 +9,16 @@
 class Config:
     """Configuration for MongoDB Charm."""
 
-    SUBSTRATE = "vm"
-    # We expect the MongoDB container to use the default ports
     MONGODB_PORT = 27017
+    SUBSTRATE = "vm"
     ENV_VAR_PATH = "/etc/environment"
     MONGODB_SNAP_DATA_DIR = "/var/snap/charmed-mongodb/current"
     MONGOD_CONF_DIR = f"{MONGODB_SNAP_DATA_DIR}/etc/mongod"
     MONGOD_CONF_FILE_PATH = f"{MONGOD_CONF_DIR}/mongod.conf"
-    SNAP_PACKAGES = [("charmed-mongodb", "5/edge", 82)]
+    SNAP_PACKAGES = [("charmed-mongodb", "5/edge", 84)]
+    CONFIG_SERVER = "config-server"
```

Comment:
Maybe we can put CONFIG_SERVER, REPLICATION and SHARD into "ROLE" subclass or something like that?